sdk/python/setup.py

```diff
@@ -32,7 +32,7 @@
     'google-auth>=1.6.1',
     'requests_toolbelt>=0.8.0',
     'cloudpickle==1.1.1',
-    'kfp-server-api==0.2.5',  #Update the upper version whenever a new version of the kfp-server-api package is released. Update the lower version when there is a breaking change in kfp-server-api, or kfp sdk depends on new api changes in kfp-server-api.
+    'kfp-server-api==0.3.0',  #Update the upper version whenever a new version of the kfp-server-api package is released. Update the lower version when there is a breaking change in kfp-server-api, or kfp sdk depends on new api changes in kfp-server-api.
```

Comment:
@Ark-kun @numerology I guess we need to discuss this one more time.

Should this be `'kfp-server-api >= 0.2.5 <= 0.3.0` instead? Because no breaking change in kfp-server-api yet.

Reply:
There is also the same requirement in `requirements.txt`. Why are they in both places?

Reply:
My understanding is, the version in `requirements.txt` is only for testing, while the version in setup.py is used when others install published `kfp` sdk python package.

@numerology for correction

Reply:
Thanks, that makes sense. I've updated the version to pin to a range instead. PTAL.

Reply:
Yep, I think that looks good to me, we shouldn't release a kfp-server-api with breaking change using patch version.